requirements.txt

```diff
@@ -1,21 +1,32 @@
 astroid==2.0.2
+atomicwrites==1.1.5
+attrs==18.1.0
+autopep8==1.3.5
 certifi==2018.4.16
 chardet==3.0.4
 coverage==4.5.1
 defusedxml==0.5.0
 Django==2.0.6
 django-cors-middleware==1.3.1
 django-extensions==2.1.0
+django-filter==2.0.0
 djangorestframework==3.8.2
+djangorestframework-jwt==1.11.0
 idna==2.7
 isort==4.3.4
 lazy-object-proxy==1.3.1
 mccabe==0.6.1
+more-itertools==4.3.0
 oauthlib==2.1.0
+pluggy==0.7.1
+psycopg2==2.7.5
```

Comment:
You could get rid of psycopg2 here since it is being deprecated in preference to psycopg2-binary which is already in our file.